distribution/windows/install.nsi

```diff
@@ -150,7 +175,8 @@ Section "!OpenRCT2" Section1
 
     SetShellVarContext all
 
-    ; Copy data files
+    ; Copy data files. Clear out the old dir first, to ensure upgrades do not result in old objects sticking around.
+    RMDir /r "$INSTDIR\data"
```

Comment:
This scares me a little, considering historically users have installed OpenRCT2 in the same folder as their base RCT2 folder. While they _shouldn't_, this would remove more than they'd bargained for.

Can we compromise and instead delete just the `data/object` folder, if one exists?